Do not run close animation when closed

lontivero opened the pull request:
Fixes https://github.com/zkSNACKs/WalletWasabi/issues/3448

lontivero commented:
Closing in favor of https://github.com/zkSNACKs/WalletWasabi/pull/3455 that contains bug fixes in Avalonia.

danwalmsley commented:
@lontivero i think you closed this because i typed the wrong issue number in https://github.com/zkSNACKs/WalletWasabi/issues/3443... sorry

molnard commented:
I cannot reproduce the original issue. @lontivero can you investigate if this PR still required?

lontivero commented:
I fixed the issue in this PR but after that @danwalmsley created PR https://github.com/zkSNACKs/WalletWasabi/pull/3455 then I verified that that PR also fixed the same issue with a newer Avalonia version that contained also other fixes so, that new version was required anyway and made my fix redundant. That's why I closed the PR, because the issue is not there anymore.
